### PR TITLE
TDP-3141: add ctrl+click management on inventory list

### PR DESCRIPTION
**Pull request by charlesnguyen**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-3141

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)

**(Optional) Other information**:

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/431/) [:x:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/431/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/442/) [:x:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/442/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/444/) [:x:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/444/)

**Comment by frassinier** [comment]
In fact, Mousewheel middle button click should be managed too

File: dataprep-webapp/src/app/services/dataset-workflow/upload-workflow-service.js [hunk 1]
```diff
@@ -60,12 +60,15 @@ export default function UploadWorkflowService($state, $window, StateService, Dat
      * @description Try to open a dataset. If it is a draft, we open the draft import wizard instead.
      * @param {object} dataset The dataset to open
      */
-	this.openDataset = function openDataset(dataset) {
+	this.openDataset = function openDataset(dataset, event) {
 		if (dataset.draft) {
```
Comment (frassinier) [inline dataprep-webapp/src/app/services/dataset-workflow/upload-workflow-service.js]: What about if it's a draft case?

Comment (charlesnguyen) [inline dataprep-webapp/src/app/services/dataset-workflow/upload-workflow-service.js]: For draft case, a popup open to choose the sheet...we can not open this in the new tab.

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/524/) [:x:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/524/)

**Pull request merged**